### PR TITLE
Rebuilt int_clearinghouse__nearby_stations

### PR DESCRIPTION
**Pull request by mmmiah**
This PR rebuilt the 'int_clearinghouse__nearby_stations' model based on 'int_vds__station_config' and named as 'int_vds__nearby_stations'. This PR also updated the ymal for 'int_vds__nearby_stations' and model connection in 'int_imputation__local_regional_regression_coefficients'. The details issues are notes in #296

### CODE REVIEW

**Review by ian-r-rose — COMMENTED** [review]
This looks great @mmmiah! Code looks good to me, I think I just see one issue with the docs

File: transform/models/intermediate/vds/_vds.yml [hunk 1]
```diff
@@ -131,3 +131,46 @@ models:
...
+      Multiple detectors across a set of lanes constitute a station.
+
+      Unlike the equivalent configuration models for controller and station,
+      this metadata model is further joined with metadata for each detector's
+      station. This adds some further denormalization to the table, but is a
+      practical response to so many downstream metrics relying on station metadata.
+
+      This table is a slowly-changing-dimension-2, meaning that it has multiple
+      different versions of the metadata for each controller, with `_valid_from`
+      and `_valid_to` datetime columns indicating when the version is correct.
```
[the remark this answers is not in the document]
Comment (mmmiah) [reply transform/models/intermediate/vds/_vds.yml]: oppps, did not update the model description. Updated now.

**Review by ian-r-rose — APPROVED** [review]
Looks good to me, thanks @mmmiah! Want to delete the `int_clearinghouse__nearby_stations` model and we can merge?

File: transform/models/intermediate/clearinghouse/_clearinghouse.yml
Comment (ian-r-rose) [inline transform/models/intermediate/clearinghouse/_clearinghouse.yml:1]: This file still had some tables that were in use!

Comment (mmmiah) [reply transform/models/intermediate/clearinghouse/_clearinghouse.yml:1]: right! Dropped only relevant table that we rebuilt.

**Review by britt-allen — APPROVED** [review]
Thank you Mintu!

**Pull request merged**